infrastructure/variable.tf

```diff
@@ -39,10 +39,16 @@ variable "zip_store_dynamodb_table_name" {
   default = "ZipStoreReferenceMetadata"
 }
 
-variable "auth_dynamodb_table_name" {
+variable "auth_state_dynamodb_table_name" {
   type    = string
   default = "AuthStateReferenceMetadata"
 }
+
+variable "auth_session_dynamodb_table_name" {
+  type    = string
+  default = "AuthSessionReferenceMetadata"
+}
+
```

Comment:
Add description="" prop where possible, this is generally the first line or two from the terraform docs for this prop

Reply:
Thanks! I added description to the variables related to this PR.